### ISSUE TITLE
Add command with --file fails if the file is already in the right place

### ISSUE THREAD

**Issue by Nagasaki45** [Nagasaki45/bibo#53]
The file is already in the right place, with some name that conflicts with bibo-default.

```
(venv) TK-MAC :: PhD/bib » bibo add --doi "https://doi.org/..." --file files/Cannon_2019.pdf
Traceback (most recent call last):
  File "/Users/thomaskaplan/libs/venv/bin/bibo", line 11, in <module>
    load_entry_point('bibo==0.0.22', 'console_scripts', 'bibo')()

...

  File "/Users/thomaskaplan/libs/venv/lib/python3.8/shutil.py", line 409, in copy
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/Users/thomaskaplan/libs/venv/lib/python3.8/shutil.py", line 239, in copyfile
    raise SameFileError("{!r} and {!r} are the same file".format(src, dst))
shutil.SameFileError: 'files/Cannon_2019.pdf' and '.../files/cannon_2019.pdf' are the same file```
```

Originally reported in #52 but I think it deserves a separate issue.

**Comment by Kappers** [comment]
I can take this one.

**Comment by Nagasaki45** [comment]
Solved in f2eeae8e214a1f01c3b6975af359c0d2acec41ad, aa64b6480977bdb94c226b70074755c82b6840d8, and 39af537f2bb1b312c0df5d381f392cf2ce939d77.
